docs/modules/usage/how-to/persist-session-data.md

```diff
@@ -0,0 +1,16 @@
+# Persisting Session Data
+
+Using the standard installation, the session data is stored in memory. Currently, if OpenHands' service is restarted,
+previous sessions become invalid (a new secret is generated) and thus not recoverable.
+
+## How to Persist Session Data
+
+### Development Workflow
```

Comment:
I got this to work with the below instructions but could not get it to work using the docker command. I ran the docker command with:
`-e FILE_STORE="local" -e FILE_STORE_DATA="<path>" -e JWT_SECRET="secretpass"`

Reply:
> -e FILE_STORE="local" -e FILE_STORE_DATA="<path>" -e JWT_SECRET="secretpass"

Isn't it FILE_STORE_DATA  ->  FILE_STORE_PATH instead?

Reply:
Yeah sorry FILE_STORE_PATH. Don't know why I wrote DATA...still didn't work though.
Maybe there is an issue when running it through the docker command? Cause the original doc only mentions the config.toml way as well.

Reply:
Sorry, of course not, if you were trying local. We need to use something else than local, or define a volume. 😢